maven-release-manager/src/main/java/org/apache/maven/shared/release/exec/InvokerMavenExecutor.java

```diff
@@ -400,10 +119,11 @@ public void executeGoals( File workingDirectory, List<String> goals, ReleaseEnvi
                     throw new MavenExecutorException( "Error executing Maven.",
                                                       invocationResult.getExecutionException() );
                 }
+
                 if ( invocationResult.getExitCode() != 0 )
                 {
                     throw new MavenExecutorException(
-                        "Maven execution failed, exit code: \'" + invocationResult.getExitCode() + "\'",
+                        "Maven execution failed, exit code: '" + invocationResult.getExitCode() + "'",
```

Comment:
I think, the single quotes are redundant: (1) it is a number, (2) we have a colon